contracts/mocks/ERC721ALowCap.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+// Creators: Chiru Labs
+
+pragma solidity ^0.8.4;
+
+import '../extensions/ERC721ALowCap.sol';
+
+contract ERC721ALowCapMock is ERC721ALowCap {
```

Comment:
File name better be the same as contract name, try changing the file name to `ERC721ALowCapMock.sol`.